### PR TITLE
Several small updates

### PR DESCRIPTION
**Pull request by michelfernandez**
- Added aes-for-biz page on sub-navigation and footer
- Added ID-Option factsheet
- Added aes-for-biz teaser on home
- Added kellerhals carrard logo on home
- Removed on vertical HR - 2019 teaser

### CODE REVIEW

File: site/content/english/about.md [hunk 1]
```diff
@@ -40,14 +40,16 @@ The remaining ten percent require a high degree of trust, for example signing em
 
 {{< content top=6 width=narrow >}}
 {{% markdown %}}
-Contracts no longer have to take a detour via printers and paper in order to be signed in a legally valid manner.
+Agreements should no longer have to take the detour via printer and paper in order to be legally signed:
```
Comment (betatim) [inline site/content/english/about.md]:
```suggestion
Agreements should no longer have to take a detour via printer and paper in order to be legally signed: 
```

File: site/content/english/about.md [hunk 2]
```diff
@@ -40,14 +40,16 @@ The remaining ten percent require a high degree of trust, for example signing em
...
 {{< content top=6 width=narrow >}}
 {{% markdown %}}
-Contracts no longer have to take a detour via printers and paper in order to be signed in a legally valid manner.
+Agreements should no longer have to take the detour via printer and paper in order to be legally signed: 
 
-With Skribble, signature processes are handled 100% digitally, building on Swiss and European standards/regulations that grant electronic signatures the same legal weight as handwritten signatures.
+Skribble is a one-stop shop for electronic signatures that makes signature processes 100% digital. Skribble provides the legally suitable electronic signature for every type of contract – even if the law requires the same legal weight as a handwritten signature.
 
-Security does not come at the expense of simplicity: every company, regardless of size and technical know-how, should be able to benefit from the advantages of this digital transformation.
+Security does not come at the expense of simplicity: every company, regardless of size and technological experience, should be able to benefit from the advantages of digital transformation. This is wyh Skribble can be used securely evene without any IT knowledge.
```
Comment (betatim) [inline site/content/english/about.md]:
```suggestion
Security does not come at the expense of simplicity: every company, regardless of size and technological experience, should be able to benefit from the advantages of digital transformation. This is why Skribble can be used securely even without any IT knowledge. 
```

File: site/content/english/about.md [hunk 3]
```diff
@@ -40,14 +40,16 @@ The remaining ten percent require a high degree of trust, for example signing em
...
+Skribble is a one-stop shop for electronic signatures that makes signature processes 100% digital. Skribble provides the legally suitable electronic signature for every type of contract – even if the law requires the same legal weight as a handwritten signature.
 
-Security does not come at the expense of simplicity: every company, regardless of size and technical know-how, should be able to benefit from the advantages of this digital transformation.
+Security does not come at the expense of simplicity: every company, regardless of size and technological experience, should be able to benefit from the advantages of digital transformation. This is wyh Skribble can be used securely evene without any IT knowledge. 
 
 ## And that's just the beginning:
-Building trust in digital business relationships should become as simple and intuitive as toasting bread. In 2018 we formed a team of Trust Shapers that has been working towards this future.
+Building trust in digital business relationships should become as simple and
```
Comment (betatim) [inline site/content/english/about.md]:
```suggestion
Building trust in digital business relationships should be as simple and
```

File: site/content/french/team.md [hunk 4]
```diff
@@ -5,11 +5,12 @@ draft: false
...
 type: team
 
+
 # trust shapers, advisors, ...
 # expects jpg and web image formats
 groups:
-  - title: Trust shapers
-    text: Les gens derrière Skribble
+  - title: Trust Shapers
+    text: The people behind Skribble
```
Comment (betatim) [inline site/content/french/team.md]: Should this be in english?

File: site/content/german/about.md [hunk 5]
```diff
@@ -41,12 +41,12 @@ Bei den übrigen zehn Prozent handelt es sich um Schritte, die ein hohes Mass an
 {{% markdown %}}
 Verträge sollen nicht mehr den Umweg über Drucker und Papier nehmen müssen, um rechtssicher unterschrieben zu werden.
 
-Mit Skribble werden Signaturprozesse 100% digital abgewickelt, basierend auf gesetzlich geregelten Standards, die die handschriftliche Unterschrift rechtsgültig und beweiskräftig ersetzen.
+Mit Skribble wickeln Firmen jeden Signaturprozess zu 100% digital ab. Als One-Stop-Shop für elektronisches Signieren liefert Skribble für jede Art von Vertrag die rechtlich passende elektronische Signatur – auch dann, wenn das Gesetz eine handschriftliche Unterschrift verlangt.  
 
-Dabei geht die Sicherheit nicht auf Kosten von Einfachheit: Jede Firma, unabhängig von Grösse und technologischem Know-how, soll von den Vorteilen der digitalen Transformation profitieren können.
+Dabei geht die Sicherheit nicht auf Kosten von Einfachheit: Jede Firma, unabhängig von Grösse und technologischem Know-how, soll von denVorteilen der digitalen Transformation profitieren können. Deshalb kann Skribble auch ohne jedes IT-Wissen sicher eingesetzt werden.
```
Comment (betatim) [inline site/content/german/about.md]:
```suggestion
Dabei geht die Sicherheit nicht auf Kosten von Einfachheit: Jede Firma, unabhängig von Grösse und technologischem Know-how, soll von den Vorteilen der digitalen Transformation profitieren können. Deshalb kann Skribble auch ohne jedes IT-Wissen sicher eingesetzt werden.
```

File: site/content/french/team.md (hunk 4 above)
Comment (michelfernandez) [reply site/content/french/team.md]: It's wrong. Thanks! I will change it in FR and DE.

**Comment by michelfernandez** [comment]
> You are adding pictures for Chantal and Barbara but they aren't mentioned in the diff of the team file. Are they already in there or are we just adding the pictures for later use?

yes, Barbara, Chantal, Maxim and Rainer were there with a blind image. Only Roland was new.

**Pull request merged**